src/components/publicApi/LoginScreen.tsx

```diff
@@ -51,6 +51,8 @@ interface Props {
    */
   fastLogin?: boolean
 
+  forceLightAccountCreate?: boolean
```

Comment:
`/** Force light account creation, even if the device has existing accounts. */`